Makefile

```diff
@@ -495,6 +505,60 @@ test-mssql-migration: migrations.mssql.test migrations.individual.mssql.test gen
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.mssql.test -test.failfast
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.individual.mssql.test -test.failfast
 
+.PHONY: test-e2e%
+# Use only file logging for end-to-end tests
+test-e2e%: TEST_LOGGER ?= file
+
+.PHONY: test-e2e
+test-e2e: test-e2e-sqlite
+
+.PHONY: test-e2e\#%
+test-e2e\#%: test-e2e-sqlite\#%
+# Kind of a hack to get makefile to accept passing arguement
+	true
+
+.PHONY: test-e2e-sqlite
+test-e2e-sqlite: TAGS+=sqlite sqlite_unlock_notify
+test-e2e-sqlite: build generate-ini-sqlite
+	npx playwright install $(PLAYWRIGHT_FLAGS)
+	GITEA_ROOT=$(CURDIR) GITEA_URL="http://localhost:3003" GITEA_EXECUTABLE=$(EXECUTABLE) GITEA_CONF=integrations/sqlite.ini ./tools/e2e/run_e2e.sh
```

Comment:
Is there a way to override the `GITEA_URL`?  This port is wrong when I run the make task.